### PR TITLE
feat: alternative layout approach using flexbox

### PR DESCRIPTION
**Pull request by billiegoose**
@casserni this one might work better. Won't have the grid-gap issue, AND you get less typing!

### CODE REVIEW

File: src/components/Form.tsx [hunk 1]
```diff
@@ -63,13 +78,62 @@ export const Form: React.FunctionComponent<IFormtronControl> = ({
...
-    </div>
-  );
+      }
+    }
+    if (cells.length > 0) {
+      innerStuff.push(<Flex>{cells}</Flex>);
+    }
+  }
+  // const innerStuff = (
+  //   <div
```
Comment (casserni) [inline src/components/Form.tsx:92]: Remove this commented out section

**Review by casserni — APPROVED** [review]
Just delete that comment section since it’s in the other PR and then good to go!

File: src/components/Form.tsx (hunk 1 above)
Comment (billiegoose) [reply src/components/Form.tsx:92]: oops
didn't see your comment in time 😊

Comment (billiegoose) [reply src/components/Form.tsx:92]: yah that's embarrassing whoops. Oh vell.

Comment (casserni) [reply src/components/Form.tsx:92]: Haha no worries I'll strp it out in my pr

**Pull request merged**